### PR TITLE
Fixed issue #112 and #55 and added slot tags

### LINKED ISSUE #28: Browserify Failed!: 'import' and 'export' may appear only with 'sourceType: module'
Hello,
I have tried
https://github.com/yuche/vue-strap/issues/19#issuecomment-147896277
but it doesn't fix the problem.
my app.js is

```
window.Vue = require('vue');
var resource = require('vue-resource');
window.Vuestrap = require('vue-strap/src/Alert.vue');
Vue.use(resource);
```
@yuche what is the solution?

### LINKED ISSUE #112: Models broken when used with vueify/browserify/elixir
When using this package with elixir it seems that the imports within the modal component cause a break.

It concerns using `export default` over `module.export`. The first causes it to fail and the second works. This could likely be a problem in vueify itself but considering there is a fix I know of for `vue-strap` I decided to post here first.

To replicate use `vue-strap` with laravel elixir `^4.0.0` and `laravel-elixir-vueify`, when using browserify to transform a file importing the modal component it will fail. The same effect should happen in any environment but that is mine, even adjusting it/ reinstalling yielded no results so I am confident it is easy to recreate.

Seems this is very close to #55

### PR DESCRIPTION
**Pull request by prog-rajkamal**
TL;DR I fixed #112 #55  #28 #13 (all are same issue) correctly and issue #133  and added some slot tags where i found appropriate

I ran into  the issue yesterday and didn't like the hack original issuer ( #28  ) used. so i traced its cause and fixed it.
 the problem was that vue-strap does not specify `transform: babelify` for browersify-transforms even though index,js and main,js are es6 code.
So I added the code. and it works.

Also I fixed a typo in tabSet.vue #133 
I changed ` <div class="tab-content" v-el:tabContent>` to `<div class="tab-content" v-el:tab-content>`

Other changes I did was to wrap `{{header}}`, `{{title}}` and `{{{ content}}}` in a `<slot>` tag so by default code will work as it used to, but now we can use `<slot>` to pass html around instead of using attributes.

### CODE REVIEW

**Comment by prog-rajkamal** [comment]
I think it should be noted that I am using ` require('vue-strap/src/index.js');` instead of ` require('vue-strap')` throws a lot of error in gulp.

**Comment by billmurrin** [comment]
+1. This fix (adding babelify to transform on index.js) worked resolving the Vueify/Browserify failures I was having. I did however, have to install babelify with npm so that it would work. 

Is it possible to have the docs reflect this or add babelify added as a dependency when getting used with Vueify/Browserify?

**Comment by yuche** [comment]
Thanks. Big help.
